charm-slurmctld/src/charm.py

```diff
@@ -551,6 +556,65 @@ def _infludb_info_action(self, event):
         logger.debug(f"## InfluxDB-info action: {influxdb_info}")
         event.set_results({"influxdb": influxdb_info})
 
+    def _on_create_user_group(self, event):
+        """Create the user and group provided."""
+        user = self._user_group._user_name
+        user_uid = self._user_group._user_uid
+        group = self._user_group._group_name
```

Comment:
This is a private variable. Can we make it public if we are using outside the class?

Reply:
yep, we can. I shall change them